HpToolsLauncher/Helper.cs

```diff
@@ -592,9 +592,13 @@ public static string GetParallelRunnerDirectory(string parallelRunnerExecutable)
 
             if (uftFolder == null) return null;
 
-            return uftFolder + @"bin\" + parallelRunnerExecutable;
+            return Path.Combine(uftFolder, "bin", parallelRunnerExecutable);
         }
 
+        /// <summary>
+        /// Why we need this? If we run jenkins in a master slave node where there is a jenkins service installed in the slave machine, we need to change the DCOM settings as follow:
+        /// dcomcnfg.exe -> My Computer -> DCOM Config -> QuickTest Professional Automation -> Identity -> and select The Interactive User
+        /// </summary>
```

Comment:
I cannot remember whether this method exists for long time or was added due to some reasons.
It seems the method attempts to change the DCOM setting automatically and ask user to do it manually if failed.
Since this is a standalone tool, it means the tool might be used on a machine without any Jenkins components.
So, it makes sense that we firstly attempt to do that automatically and if failed then ask user to do it manually.

Reply:
This topic is pretty advanced and all I know is that DCOM settgins are required in order to enable UFT to be run remotely:
https://support.microfocus.com/kb/kmdoc.php?id=KM02239325
For a deeper investigation I think a dedicated task would be recommended.
In this PR, I just removed the duplicate methods ChangeDCOMSettingToInteractiveUser and GetQuickTestProfessionalAutomationRegKey from GuiTestRunner.cs, but I copied the summary into Helper.cs (on ChangeDCOMSettingToInteractiveUser method header).

Reply:
Ok, got you